src/rpc/mempool.cpp

```diff
@@ -249,34 +249,81 @@ static RPCHelpMan testmempoolaccept()
+            RPCResult{RPCResult::Type::NUM, "descendantcount", "number of in-mempool descendant transactions (including this one)"},
+            RPCResult{RPCResult::Type::NUM, "descendantsize", "virtual transaction size of in-mempool descendants (including this one)"},
+            RPCResult{RPCResult::Type::NUM, "ancestorcount", "number of in-mempool ancestor transactions (including this one)"},
+            RPCResult{RPCResult::Type::NUM, "ancestorsize", "virtual transaction size of in-mempool ancestors (including this one)"},
+            RPCResult{RPCResult::Type::NUM, "chunksize", "virtual transaction size of this transaction's chunk"},
+            RPCResult{RPCResult::Type::NUM, "clusterid", "id of the cluster containing this tx"},
+            RPCResult{RPCResult::Type::STR_HEX, "wtxid", "hash of serialized transaction, including witness data"},
+            RPCResult{RPCResult::Type::OBJ, "fees", "",
+                {
+                    RPCResult{RPCResult::Type::STR_AMOUNT, "base", "transaction fee, denominated in " + CURRENCY_UNIT},
+                    RPCResult{RPCResult::Type::STR_AMOUNT, "modified", "transaction fee with fee deltas used for mining priority, denominated in " + CURRENCY_UNIT},
+                    RPCResult{RPCResult::Type::STR_AMOUNT, "ancestor", "transaction fees of in-mempool ancestors (including this one) with fee deltas used for mining priority, denominated in " + CURRENCY_UNIT},
+                    RPCResult{RPCResult::Type::STR_AMOUNT, "descendant", "transaction fees of in-mempool descendants (including this one) with fee deltas used for mining priority, denominated in " + CURRENCY_UNIT},
+                    RPCResult{RPCResult::Type::STR_AMOUNT, "chunk", "transaction fees of chunk, denominated in " + CURRENCY_UNIT},
+                }},
+            RPCResult{RPCResult::Type::ARR, "depends", "unconfirmed transactions used as inputs for this transaction",
+                {RPCResult{RPCResult::Type::STR_HEX, "transactionid", "parent transaction id"}}},
+            RPCResult{RPCResult::Type::ARR, "spentby", "unconfirmed transactions spending outputs from this transaction",
+                {RPCResult{RPCResult::Type::STR_HEX, "transactionid", "child transaction id"}}},
+            RPCResult{RPCResult::Type::BOOL, "bip125-replaceable", "Whether this transaction signals BIP125 replaceability or has an unconfirmed ancestor signaling BIP125 replaceability.\n"},
+            RPCResult{RPCResult::Type::BOOL, "unbroadcast", "Whether this transaction is currently unbroadcast (initial broadcast not yet acknowledged by any peers)"},
     };
 }
 
+static void clusterToJSON(const CTxMemPool& pool, UniValue& info, const Cluster& c) EXCLUSIVE_LOCKS_REQUIRED(pool.cs)
+{
+    AssertLockHeld(pool.cs);
+    info.pushKV("vsize", (int)c.m_tx_size);
+    info.pushKV("txcount", (int)c.m_tx_count);
+    info.pushKV("clusterid", (int)c.m_id);
+    UniValue chunks(UniValue::VARR);
+    for (auto &chunk : c.m_chunks) {
```

Comment:
0b284b5fd29c06d46c1ec60ea7e1bcd07f36feb1: can you order chunks by mining score?

Reply:
Chunks for a given cluster are already sorted in descending feerate order, is that what you're asking about or is there another issue I'm overlooking?